ros_ws/src/car_tf/src/tf_laserscan_to_pointcloud.cpp

```diff
@@ -0,0 +1,35 @@
+#include <ros/ros.h>
+#include <tf/transform_listener.h>
+#include <laser_geometry/laser_geometry.h>
+
+class LaserscanTransformer {
+     public:
+        LaserscanTransformer();
+        void scanCallback(const sensor_msgs::LaserScan::ConstPtr& laserscan);
+     private:
+        ros::NodeHandle node;
+        laser_geometry::LaserProjection projector;
+        tf::TransformListener listener;
+        ros::Publisher pointcloud_publisher;
+        ros::Subscriber laserscan_subscriber;
+};
+
+LaserscanTransformer::LaserscanTransformer(){
+        laserscan_subscriber = node.subscribe<sensor_msgs::LaserScan> ("/racer/laser/scan", 100, &LaserscanTransformer::scanCallback, this);
```

Comment:
Please create constexpressions for the topic names, ie `constexpr const char* TOPIC_LASER_SCAN = "/racer/laser/scan"`.